src/AzureAppConfigurationImpl.ts

```diff
@@ -44,7 +44,10 @@ export class AzureAppConfigurationImpl extends Map<string, unknown> implements A
 
     public async load() {
         const keyValues: [key: string, value: unknown][] = [];
-        const selectors = this.options?.selectors ?? [{ keyFilter: KeyFilter.Any, labelFilter: LabelFilter.Null }];
+
+        // validate selectors
+        const selectors = validatedSelectors(this.options?.selectors);

```

Comment:
`validatedSelectors` sounds more like a variable name than a method name.
What do you think about calling it `getValidSelectors`?